### ISSUE TITLE
Stripey rendering artifacts on large bodies of ice or water viewed from far away

### ISSUE THREAD

**Issue by Majora320** [GTNewHorizons/Angelica#434]
### Your GTNH Discord Username

majora320

### Mod Version

1.0.0-alpha36a

### Java Version

Java 17

### Graphics Card Vendor

AMD/ATI

### Bug Report

When viewing deep bodies of ice or water (the issue doesn't occur with shallow water) from far away/high up, rendering artifacts will appear in characteristic stripes. Here is an example on a large ice field:

![2024-04-20_01 35 02](https://github.com/GTNewHorizons/Angelica/assets/16787503/f90354b2-484e-479f-ab9c-01e6538a2442)

I also came across a similar issue in which grass would appear discolored in HBM: NTM's crater biome in similar striped patterns, but I'm not sure if it was related or not - if it's still present after this is fixed I'll submit another ticket.

### Mod List or GTNH Pack Version

GTNH 2.6.0-beta3

### Final Checklist

- [X] I have searched the issues and haven't found a similar issue.
- [X] I have read the known incompatibilities and this is not related to one of those.
- [X] I am running an officially released version.  (Or, if I've compiled it myself I plan to fix the issue)
- [X] This issue is not related to a feature that is disabed by default - Shaders, MCPF, etc.  [They'll be enabled when they're ready for testing]

**Comment by ah-OOG-ah** [comment]
Is translucency sorting enabled in Video Settings/Advanced?

**Comment by Majora320** [comment]
Yes, and disabling translucency sorting does appear to fix it - though that does cause some other rendering issues with bits of underwater oil columns being rendered on top of the water.
